src/components/global/ModalsStudent.tsx

```diff
@@ -0,0 +1,99 @@
+// import CloseIcon from '@mui/icons-material/Close'
+import OpenInNewIcon from '@mui/icons-material/OpenInNew'
+import { Box, Button, IconButton, Modal, Typography } from '@mui/material'
+import { useState } from 'react'
+import { Classroom, Player, Repeatable, TaskWithStatus } from '../../types'
+import { completeRepeatable, completeTask } from '../../utils/mutations'
+import { format, fromUnixTime } from 'date-fns'
+import { StudentTaskModalBox, ModalTitle, StudentBoxInModal } from '../global/TaskModalStyles'
+
+interface PropsTask {
+	classroom: Classroom
+	player: Player
+	task: TaskWithStatus
+	type: 'task'
+}
+
+interface PropsRepeatables {
+	classroom: Classroom
+	player: Player
+	task: Repeatable
```

Comment:
I think this prop should be named `repeatable` instead

Reply:
Alternatively, I know our language of "tasks" vs "repeatables" can be a bit confusing, so we can have a conversation on how to standardize how we name one-time tasks vs repeatable tasks throughout the code